app/components/ListQuestions/ListQuestions.jsx

```diff
@@ -77,6 +80,21 @@ const ListQuestions = ({
 
 
   const renderQuestionsList = () => {
+    const onLikeButtonClick = async (questionId) => {
```

Comment:
Async is required? I don't see any await inside function

Reply:
You are totally right about both functions not being asynchronous, I had forgotten to remove the keyword, I've removed it now, thank you!